src/services/expenses.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+const expenses = [];
+
+let countExpensesId = 1;
+
+const getExpenses = ({ from, to, category, userId }) => {
+  const startFrom = Date.parse(from);
+  const end = Date.parse(to);
+  let filteredExpenses = expenses;
+
+  if (startFrom) {
+    const result = filteredExpenses.filter(expense => {
```

Comment:
I think there is no need to create variable for result, you can write it to `filteredEpxenses`